pages/common/node.md

```diff
@@ -13,3 +13,7 @@
 - Evaluate JavaScript by passing it in the command:
 
 `node -e "{{code}}"`
+
+- Evaluate and print result, useful to see node's dependencies versions:
+
+`node -p {{process.versions}}`
```

Comment:
Could we enclose `{{process.versions}}` in single quotes please?

Passing code as a CLI argument may potentially include special characters, which we want to ensure get passed correctly.

Reply:
done